.github/ISSUE_TEMPLATE/bug report.md

```diff
@@ -0,0 +1,20 @@
+<!-- Please submit an issue as per the template, not doing so may result in your issue not being addressed in a timely manner. Thanks!-->
```

Comment:
nit: I think "Please use this template when reporting a bug. Not {...} " might be clearer. I think it's good to avoid using the word "issue" since that can mean more things than "bug", as in "feature issue" "cleanup issue" etc.